hushline/model.py

```diff
@@ -188,6 +225,25 @@ class User(Model):
     )
     smtp_sender: Mapped[Optional[str]]
 
+    # Paid tier fields
+    tier_id: Mapped[int | None] = mapped_column(db.ForeignKey("tiers.id"), nullable=True)
+    tier: Mapped["Tier"] = relationship(backref=db.backref("tiers", lazy=True))
+
+    stripe_customer_id = mapped_column(db.String(255))
+    stripe_subscription_id = mapped_column(db.String(255), nullable=True)
+    stripe_subscription_cancel_at_period_end = mapped_column(
+        db.Boolean, default=False, nullable=True
```

Comment:
We have `default=False` but also `nullable=True` which might mean we never have `null` value set. Do we ever need this to be `null`? Or can we force it to be always false? Also slight preference for `server_default=text('false')`.

Reply:
Fixed in 40b1ca102ea381b325f06f6a1ec5ceb61c4228a2